### ISSUE TITLE
Error adding your authenticator

### ISSUE THREAD

**Issue by AndriiSeeker** [Jessecar96/SteamDesktopAuthenticator#967]
I removed my authenticator, because of the mistake (https://github.com/Jessecar96/SteamDesktopAuthenticator/issues/954).
I am trying to add it again, but I get another error.
I accept my login with email.
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/59fb3f83-f48f-4078-a842-c60408982dc9)
But after clock "ok" got error.
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/fdcd5455-97ee-4c76-b449-dd485817570c)

**Comment by merelenten** [comment]
Link the phone number to the account first, then you can link the authenticator using this version of sda.

**Comment by AndriiSeeker** [comment]
My phone number is linked
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/e20cfdf3-db68-4ae8-a1a1-19cc27d1e4af)

**Comment by merelenten** [comment]
Check if your steam guard is completely disabled (the protection icon should be red).
But I want to note that this method of binding in sda is very selective for accounts. It always works well only with an account that has never been linked to an authenticator or phone number.

**Comment by dyc3** [comment]
No, the steam guard icon does **not** need to be red. SDA is just very broken right now.

**Comment by K-Paks** [comment]
Facing similar issues. Is there an older version of SDA that works?

**Comment by dyc3** [comment]
Somebody was posting that setting up the authenticator works on 1.0.11, and then upgrade to 1.0.13 for confirmations to work. Idk, something like that, haven't tried it.

If you are interested, I maintain steamguard-cli. It's capable of reading the exactly the same maFiles that SDA does, so you don't have to deal with a trade ban. You can download it here: https://github.com/dyc3/steamguard-cli/releases

Read the quickstart here: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md

Alternatively, some people have made community patches (#859), and there's other software that does the same thing, if you feel comfortable with that kind of thing. Otherwise, you can transfer the authenticator to the official steam mobile app, but that does cause a 2 day trade ban.

**Comment by ForDownloadOnly** [comment]
> Facing similar issues. Is there an older version of SDA that works?
> 
> [UPDATE] Downloaded the pre-release 1.0.14 version and it worked

did you add the mobile number on steam 1st before adding it on SDA?

**Comment by dyc3** [comment]
Steam doesn't let you add an authenticator before you add a phone number. I don't remember if SDA prompts you to do that, but it shouldn't matter if you add a phone number from SDA or in Steam directly.
